sqe-api-server/Serialization/ArtefactConversion.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections.Generic;
+using System.Linq;
+using SQE.API.DTO;
+using SQE.DatabaseAccess.Models;
+
+namespace SQE.API.Server.Serialization
+{
+    public static partial class ExtensionsDTO
+    {
+        public static ArtefactDTO ToDTO(this ArtefactModel artefact, uint editionId)
+        {
+            return new ArtefactDTO
+            {
+                id = artefact.ArtefactId,
+                editionId = editionId,
+                imageId = artefact.ImageId,
+                artefactDataEditorId = artefact.ArtefactDataEditorId,
+                mask = artefact.Mask,
+                artefactMaskEditorId = artefact.MaskEditorId,
+                isPlaced = artefact.Rotate.HasValue,
```

Comment:
I think `isPlaced` should be true if the translation is defined. We can have a rotation even if the artefact hasn't been placed (if we rotate it in the artefact editor)

Reply:
Oops! you are right.